course_discovery/apps/course_metadata/search_indexes.py

```diff
@@ -392,6 +392,10 @@ class PersonIndex(BaseIndex, indexes.Indexable):
     get_profile_image_url = indexes.CharField(model_attr='get_profile_image_url', null=True)
     position = indexes.MultiValueField()
     organizations = indexes.MultiValueField(faceted=True)
+    marketing_id = indexes.IntegerField(model_attr='marketing_id')
+    marketing_url = indexes.CharField(model_attr='marketing_url')
+    designation = indexes.CharField(model_attr='designation')
```

Comment:
`designation` -> `marketing_designation`

Reply:
Regarding using marketing_designation we recently introduced this for our marketing needs. In model, we use field name designation that’s why I preferred using the same in SearchIndex to stay consistent.
A way of changing it would be changing it in the model as well + everywhere. 
What do you say?

Reply:
From slack conversation, we are sticking with **designation** for now.
https://edlyinc.slack.com/archives/C010MM9C282/p1621850534032900?thread_ts=1621845483.025900&cid=C010MM9C282